src/frontend/jest/config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
replace `jest` config in `package.json` to its own file and update the content to replace Create React App specific settings